cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -249,6 +249,12 @@
     # nvidia handling, glob needs /usr/** and the launcher must be
     # able to bind mount the nvidia dir
     /sys/module/nvidia/version r,
+    /sys/**/drivers/nvidia{,_*}/* r,
+    /sys/**/nvidia*/uevent r,
+    /sys/module/nvidia{,_*}/* r,
```

Comment:
This covers the first rule (/sys/module/nvidia/version r,) so you can omit the first rule.

Reply:
Let's remove this in the post-mortem move and let mvo release the point release.